src/App.scss

```diff
@@ -25,24 +29,32 @@
     line-height: 1.5;
     max-width: 50ch;
     align-self: center;
+    text-align: justify;
+  }
+  
+  a {
+    text-decoration: initial;
   }
+  
 }
 
 .logo {
   cursor: pointer;
-  pointer-events: none;
+  z-index: 2;
 }
 
 .button {
```

Comment:
Takie style powinny znajdowac sie w komponencie Button. Jezeli go nie masz to pownienes taki stworzyć